Description: Remove dataclasses
 This is a temporary patch to avoid a bug in dh-python.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2021-12-22

--- nova-24.0.0.orig/requirements.txt
+++ nova-24.0.0/requirements.txt
@@ -66,5 +66,4 @@ python-dateutil>=2.7.0 # BSD
 zVMCloudConnector>=1.3.0;sys_platform!='win32'  # Apache 2.0 License
 futurist>=1.8.0 # Apache-2.0
 openstacksdk>=0.35.0 # Apache-2.0
-dataclasses>=0.7;python_version=='3.6'  # Apache 2.0 License
 PyYAML>=5.1 # MIT
